src/Task/Lock/Exception/LockConflictException.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of php-task library.
+ *
+ * (c) php-task
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Task\Lock\Exception;
+
+/**
+ * Will be thrown when a conflict is detected:
+ *  + Already acquired key was acquired.
+ *  + Not acquired key was released.
+ *  + Not acquired key was refreshed.
```

Comment:
Can't you throw more specific exception instead of one for all cases? This way it will be harder to debug as it have to (You can't directly see from the exception if the key was already acquired or not).